chronograf.go

```diff
@@ -32,12 +33,13 @@ func (e Error) Error() string {
 type Logger interface {
 	Debug(...interface{})
 	Info(...interface{})
-	Warn(...interface{})
```

Comment:
Would we never want to warn of some condition? Maybe like, "unreachable data node, trying next node" or something?